src/Controllers/GagePageController.ts

```diff
@@ -966,144 +1119,288 @@ module StreamStats.Controllers {
+                        let discharge = this.x;
+                        let stage = this.y;
+                        let peakDate = this.date
+                        return '<br>Date <b>' + peakDate +  '<br>Peak <b>' + discharge + ' cfs' + '<br>at stage <b>' + stage + ' ft'
                     }
                 }
-            }
-            function isLeapYear(year) {
-                if (year % 400 === 0) return true;
-                if (year % 100 === 0) return false;
-                return year % 4 === 0;
-            }
-            this.heatChartConfig = {
-                chart: {
-                        height: 450,
-                        width: 800,
-                        zooming: {
-                            type: 'xy'
-                        }
-                },
-                title: {
-                    text: 'Daily Streamflow',
-                    align: 'center'
-                },
-                subtitle: {
-                    text: 'Click and drag in the plot area to zoom in',
-                    align: 'center'
-                },
-                xAxis: {
-                    type: null,
-                    min: 275,
-                    max: 665,
-                    tickPositions: [275, 306, 336, 367, 398, 427, 458, 488, 519, 549, 580, 611, 650],
-                    title: {
-                        text: 'Day of Year'
-                    },
-                    threshold: 273,
-                    labels: {
-                        formatter: function() {
-                            if (this.value > 366) {
-                                this.value -= 365
-                            }
-                            if(this.value == 285) return 'Annual Average';
-                            return moment("2015 "+this.value, "YYYY DDD").format("MMM");
-                        }
+            },
+            turboThreshold: 0, 
+            type    : 'scatter',
+            color   : 'black',
+            data    : this.formattedDischargePeakDates,
+            marker: {
+                symbol: 'circle',
+                radius: 3
+            },
+            showInLegend: this.formattedDischargePeakDates.length > 0
+        },
+        {
+            name    : 'USGS Measured',
+            showInNavigator: false,
+            tooltip: { headerFormat:'<b>USGS Measured</b>',
+                pointFormatter: function(){
+                    if (this.measuredObj !== null){
+                        let dateTime = this.dateTime;
+                        let timeZone = this.timeZone;
+                        let quality = this.quality;
+                        let control = this.control;
+                        let discharge = this.x;
+                        let stage = this.y;
+                        return '<br><b>' + dateTime + ' ' + timeZone + '<br>Gage Height: <b>' + stage + ' ft' + '<br>Discharge: <b>' + discharge + ' cfs' + '<br>Quality: <b>' + quality + '<br>Control: <b>' + control
```

Comment:
Making some stylistic changes to be more consistent with pop-ups in other plots.
```suggestion
                        return '<br>Date: <b>' + dateTime + ' ' + timeZone + '</b><br>Gage Height: <b>' + stage + ' ft</b><br>Discharge: <b>' + discharge + ' cfs</b><br>Quality: <b>' + quality + '</b><br>Control: <b>' + control + '</b>'
```